cv_next/app/logout/page.tsx

```diff
@@ -0,0 +1,16 @@
+"use client"
+import { redirect } from "next/navigation"
+import { Button } from "@/components/ui/button"
+import { useSupabase } from "@/hooks/supabaseHooks"
+export default function Page() {
+  const supabase = useSupabase()
+  supabase.auth.signOut()
+
+  redirect("/")
```

Comment:
I remember that I already commented on this? Why create a component with render code if you force-redirect on first render anyway?

Reply:
This was made when the Signout was as a "link" in the navbar.
Now this page is accessed from a button in the user profile component which again, redirects to this as only the supabase things are needed. Should the functionality also be a "link" that just doesn't redirect?

Reply:
Moved the signout directly to the popup functionality as this is the referencing place